project/NativeImage.scala

```diff
@@ -11,14 +13,14 @@ object NativeImage {
         val log      = state.value.log
         val javaHome = System.getProperty("java.home")
         val nativeImagePath =
-          if (sys.props("os.name").contains("Windows"))
+          if (isWindows)
             s"$javaHome\\bin\\native-image.cmd"
           else s"$javaHome/bin/native-image"
         val classPath =
           (Runtime / fullClasspath).value.files.mkString(File.pathSeparator)
```

Comment:
Could this task be enhanced to install native image if it's missing? That way we could avoid the awkwardeness on CI?

Reply:
We may not want to install *external* dependencies on user's system when running sbt tasks. After a short discussion, we decided to stay with the current solution.